include/defines.h

```diff
@@ -301,6 +303,12 @@ enum serialRxTxEn_e {serialRxTx_RxTxDisable, serialRxTx_TxEn, serialRxTx_RxEn};
 #define ID_PARAM_BTDEV_DEACTIVATE                126
 
 #define ID_PARAM_BMS_BALUE_ADJUSTMENTS_SOC100_CELL_VOLTAGE 127
+#define ID_PARAM_BMS_BALUE_ADJUSTMENTS_SOC0_CELL_VOLTAGE 128
+
+#define ID_PARAM_TEMP_ALARM_TEMP_QUELLE          129
+#define ID_PARAM_TEMP_ALARM_BMS_QUELLE           130
+
+#define ID_PARAM_BMS_BALUE_ADJUSTMENTS_SOC0_CELL_VOLTAGE 128
 
```

Comment:
The parameters ID_PARAM_TEMP_ALARM_TEMP_QUELLE and ID_PARAM_TEMP_ALARM_BMS_QUELLE originally had the IDs 128 and 129. If existing IDs are changed, the settings are incorrect at the user. Therefore, please do not change any IDs. 
The new parameter ID_PARAM_BMS_BALUE_ADJUSTMENTS_SOC0_CELL_VOLTAGE should therefore have the ID 130.

In addition, the parameter ID_PARAM_BMS_BALUE_ADJUSTMENTS_SOC0_CELL_VOLTAGE is duplicated.

Maybe the problem was caused by my last commit.